Tests/FunctionalTests/IdentityAdIDTests.swift

```diff
@@ -26,144 +26,137 @@ class IdentityAdIDTests: XCTestCase {
         identity = Identity(runtime: mockRuntime)
         identity.onRegistered()
     }
-
-    // MARK: handleIdentityRequest
-    /// Test ad ID is updated from old to new valid value, and consent true is dispatched
-    func testGenericIdentityRequest_whenValidAdId_thenNewValidAdId() {
+    
+    // MARK: Test helpers
+    /// Sets up the IdentityProperties with the desired advertising identifier
+    func setupIdentity(withAdID: String?) -> String {
```

Comment:
We usually mark helper methods with private and place them at the bottom of the file. Also should we split this into two helper methods? 
private func setupIdentity(withAdID: String?) { }
private func getECID() -> String { }
...

Reply:
Thanks, I've relocated the method and changed access to private; also split the function logic into two like you suggested and updated usages in the test cases